cli/graphql.md

```diff
@@ -750,6 +763,69 @@ mutation CreateDraft {
 }
 ```
 
+#### Field Level Authorization
+
+The `@auth` directive specifies that a access to a specific field should be restricted
```

Comment:
The `@auth` directive specifies that a access to a specific  ..... change to --> "specifies that access to a specific".

Reply:
fixed.